cmake/tracy.cmake

```
@@ -66,7 +66,7 @@ ExternalProject_Add(
     INSTALL_COMMAND
         cp ${TRACY_HOME}/csvexport/build/unix/csvexport-release .
     BUILD_COMMAND
-        cd ${TRACY_HOME}/csvexport/build/unix && CXX=g++ TRACY_NO_LTO=1 make -f
+        cd ${TRACY_HOME}/csvexport/build/unix && CXX=g++ TRACY_NO_LTO=1 make -j -f
```

Make does not handle `-j` gracefully -- it will spawn everything in parallel with no regard to CPU count or system load.  On a small system that can be damaging.


```suggestion
include(ProcessorCount)
PocessorCount(numProcs)
...
        cd ${TRACY_HOME}/csvexport/build/unix && CXX=g++ TRACY_NO_LTO=1 make -j ${numProcs} -f
```

... and also on line 87.